### PR TITLE
Created CONTRIBUTING guide

### PR DESCRIPTION
**Pull request by rennerocha**

### CODE REVIEW

**Comment by manycoding** [comment]
Isn't it too complex?

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/scrapinghub/spidermon/issues.
+
+If you are reporting a bug, please include:
```
Comment (manycoding) [inline CONTRIBUTING.rst]: How about making an issue template and removing the list from here?

Comment (rennerocha) [reply CONTRIBUTING.rst]: I like the idea of issues templates, but I also want to have some guidance for new contributors as soon as possible. I use other scrapinghub projects as reference for this file, so what do you think creating this first version and then we discuss issue template in a different PR?

**Comment by muzaffaryousaf** [comment]
LGTM 👍

**Pull request merged**